index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homepage</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
```

Comment:
no need to add `type` for css, if you check the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) and search for `text/css` you'll notice this reference
> given that CSS is the only stylesheet language used on the web, not only is it possible to omit the type attribute, but is actually now recommended practice.